CONTRIBUTING.md

```diff
@@ -13,13 +13,130 @@ resources to make it easier to get your contribution accepted.
 
 This is a rough outline of what a contributor's workflow looks like:
 
-- Create a topic branch from where you want to base your work. This is usually master.
-- Make commits of logical units and add test case if the change fixes a bug or adds new functionality.
+### Step 1: Fork in the cloud
+
+1. Visit [https://github.com/pingcap/tikv](https://github.com/pingcap/tikv)
+2. Click Fork button (top right) to establish a cloud-based fork.
+
+### Step 2: Clone fork to local storage
+
+Define a local working directory:
+
+```sh
+# Define a local working directory:
+$ working_dir=/.../src/github.com/pingcap
+$ user={your github profile name}
+```
+
+Create your clone:
+
+```sh
+$ mkdir -p $working_dir
+$ cd $working_dir
+$ git clone https://github.com/$user/tikv.git
+$ cd $working_dir/tikv
+```
+
+View the remote info:
```

Comment:
In this code block you add a remote, people may see this comment and assume it is only diagnostic. Please split the chunks or make it more clear.